gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -39,6 +40,73 @@ class FwGenericConfigDriver(object):
     def __init__(self):
         self.timeout = cfg.CONF.rest_timeout
 
+    def _configure_log_forwarding(self, url, mgmt_ip, port):
+        """ Configures Visibility VM IP address in Service VMs to
+            configure log forwarding to the Visibility VM.
+
+            :param url: url format that is used to invoke the Service VM API
+            :param mgmt_ip: management IP of hte Service VM
+            :param port: port that is listened to by the Service VM agent
+
+            Returns: SUCCESS/Error msg
+
+        """
+
+        url = url % (mgmt_ip, port, 'configure-rsyslog-as-client')
+
+        visibility_vm_ip_address = cfg.CONF.visibility_vm_ip_address
+        if not visibility_vm_ip_address:
+            msg = ("Visibility VM IP address not configured for "
+                   "log forwarding for service at %s." % mgmt_ip)
+            LOG.info(msg)
+            return common_const.UNHANDLED
+
+        data = dict(
+                server_ip=visibility_vm_ip_address,
+                server_port=cfg.CONF.visibility_port,
+                log_level=cfg.CONF.log_level)
+        data = jsonutils.dumps(data)
+
+        msg = ("Initiating POST request to configure log forwarding "
+               "to the Visibility VM %s for service at: %r" %
+               (visibility_vm_ip_address, mgmt_ip))
+        LOG.info(msg)
+        try:
+            resp = requests.post(url, data, timeout=self.timeout)
+        except requests.exceptions.ConnectionError as err:
+            msg = ("Failed to establish connection to service at: "
```

Comment:
This log is not giving the context that things failed during log forwarding configuration.